src/core/show-status.c

```diff
@@ -28,6 +28,19 @@ int parse_show_status(const char *v, ShowStatus *ret) {
         return 0;
 }
 
+const char* show_status_to_string(ShowStatus s) {
+        switch (s) {
+        case SHOW_STATUS_YES:
+                return "yes";
+        case SHOW_STATUS_NO:
+                return "no";
+        case SHOW_STATUS_AUTO:
+                return "auto";
```

Comment:
there's also SHOW_STATUS_TEMPORARY...

The enum isn't particularly normalized, so we need to take some extra care here with making sure that the stuff we serialize/deserialize can properly reflect all possible values but from user configuration we only accept the subset that is OK...